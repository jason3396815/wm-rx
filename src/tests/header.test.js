import { h, Component } from 'preact';
import Header from '../components/Header';

describe('<Header />', () => {
	it('should render a title and nav', () => {
		expect(<Header />).toMatchSnapshot();
	});
});
