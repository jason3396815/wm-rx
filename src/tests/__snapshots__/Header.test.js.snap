// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render a title and nav 1`] = `
e {
  "attributes": undefined,
  "children": Array [],
  "key": undefined,
  "nodeName": [Function],
}
`;
